src/directives/unsafe-html.ts

```diff
@@ -39,23 +41,19 @@ export const unsafeHTML = directive((value: unknown) => (part: Part): void => {
     throw new Error('unsafeHTML can only be used in text bindings');
   }
 
-  // We don't support `unsafeHTML` on the server because there's no way to
-  // create a TemplateResult to pass to setValue using non-literal strings
-  // without coercing types (`html` only accepts a `TemplateStringsArray`).
-  if (part.isServerRendering) {
-    throw new Error('unsafeHTML does not support SSR');
-  }
-
   const previousValue = previousValues.get(part);
 
+  let result;
   if (previousValue !== undefined && isPrimitive(value) &&
-      value === previousValue.value && part.value === previousValue.fragment) {
-    return;
+      value === previousValue.value) {
+    result = previousValue.result;
+  } else {
+    // DO NOT COPY THIS CODE. IT IS UNSAFE TO CONSTRUCT TEMPLATE RESULTS THIS
+    // WAY (HENCE THE NAME).
```

Comment:
😬

We need a big todo/issue to look into how to preserve some of the security properties of client-side rendering. Looks like this implementation will pick that up since it's passing the fake TemplateResult through the normal Part API.

Reply:
Added issue: https://github.com/PolymerLabs/lit-ssr/issues/67